mautic/mautic
# [Enhancement] Allow filtering contacts by UTM data for segments.

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Yes
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | #5885
| BC breaks? |
| Deprecations? | 

#### Description:
See #5885

#### Steps to test this PR:
1. Ensure you have a contact with a unique UTM Source.
2. Create a segment to filter based on the UTM Source value provided.
3. Trigger `console mautic:segments:update` or wait for your typical cron tasks to run.
4. Check that your contact is now in the new segment, and none others are.

![UTM Filtering in Segment view](https://i.imgur.com/kpZgEGs.png)
